benchmarks/Benchmarks.Trace/AspNetCoreBenchmark.cs

```diff
@@ -25,8 +25,7 @@ public class AspNetCoreBenchmark
 
         static AspNetCoreBenchmark()
```

Comment:
Would setting the warmup count property outlined here help counter the first request issue you're trying to solve here? https://benchmarkdotnet.org/articles/guides/choosing-run-strategy.html

Reply:
Unfortunately it changes the number of times the "warmup" step will be run, but not the number of invocations per step.

After further research, this is a known issue: https://github.com/dotnet/BenchmarkDotNet/issues/994
which seems to occur for applications emitting dynamic IL (like us): https://github.com/dotnet/BenchmarkDotNet/issues/837

The recommended workaround is to run an iteration of the benchmark in the setup, so I'll stick to that solution 🙂